packages/manager/modules/pci/src/projects/creating/constants.js

```diff
@@ -9,36 +9,53 @@ import stockageSnapshotVolume from './assets/Hero16-9_Stockage_SnapshotVolume.pn
 
 export const GUIDE_URLS = {
   DEFAULT:
-    'https://help.ovhcloud.com/csm/en-ie-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050387',
-  DE:
-    'https://help.ovhcloud.com/csm/de-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050383',
+    'https://help.ovhcloud.com/csm/en-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050390',
   ASIA:
     'https://help.ovhcloud.com/csm/asia-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050384',
   AU:
     'https://help.ovhcloud.com/csm/en-au-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0038069',
-  GB:
-    'https://help.ovhcloud.com/csm/en-gb-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050388',
-  IE:
-    'https://help.ovhcloud.com/csm/en-ie-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050387',
-  SG:
-    'https://help.ovhcloud.com/csm/en-sg-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050393',
-  ES:
-    'https://help.ovhcloud.com/csm/es-es-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050389',
   CA:
     'https://help.ovhcloud.com/csm/en-ca-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050398',
-  QC:
-    'https://help.ovhcloud.com/csm/fr-ca-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050397',
+  CZ: undefined,
+  DE:
+    'https://help.ovhcloud.com/csm/de-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050383',
+  ES:
+    'https://help.ovhcloud.com/csm/es-es-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050389',
+  EU:
+    'https://help.ovhcloud.com/csm/en-ie-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050387',
+  FI: undefined,
   FR:
     'https://help.ovhcloud.com/csm/fr-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050407',
+  GB:
+    'https://help.ovhcloud.com/csm/en-gb-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050388',
+  IE:
+    'https://help.ovhcloud.com/csm/en-ie-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050387',
+  IN:
+    'https://help.ovhcloud.com/csm/asia-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050384',
   IT:
     'https://help.ovhcloud.com/csm/it-public-cloud-compute-essential-information?id=kb_article_view&sysparm_article=KB0050404',
+  LT: undefined,
```

Comment:
for LT, the displayed documentation should be the default one. You can remove this line.